pages/hub/lists/index.tsx

```diff
@@ -87,11 +132,27 @@ const ListsHub: WithPageLayout = () => {
           ))
         ) : (
           <div className="flex flex-col items-center justify-center w-full gap-4 ">
-            {!isLoading && <Title className="text-2xl">You currently have no lists</Title>}
+            {!isLoading && sessionToken ? <Title className="text-2xl">You currently have no lists</Title> : null}
           </div>
         )}
 
-        {isLoading && <SkeletonWrapper count={3} classNames="w-full" height={95} radius={10} />}
+        {isLoading && sessionToken ? <SkeletonWrapper count={3} classNames="w-full" height={95} radius={10} /> : null}
+
+        {!sessionToken && featuredLists.length > 0
+          ? featuredLists.map((list, i) => (
```

Comment:
Refactor (blocking): The server-side already checks for the session and if it doesn't exist, it populate the featuredLists. And if there is a session, the featured list is an empty array.

```suggestion
        {featuredLists.map((list, i) => (
```